index.html

```diff
@@ -12,25 +12,30 @@
 </head>
 <body>
     <div class="menu">
-        <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
-        <h1>Your forecast app</h1>
-        <div class="search">
-            <span class="enter">enter your city
-                <div class="search__icon"><img src="icons/search.png" alt=""></div>
-            </span>
+        <div class="block__menu">
```

Comment:
If you decided to use a BEM notation, then you do it wrongly. This class will be a block in BEM. You can name it `mainScreen`...